packages/aws-cdk/test/integ/cli/aws-helpers.ts

```diff
@@ -0,0 +1,143 @@
+import * as AWS from 'aws-sdk';
+import { log } from './cdk-helpers';
+
+interface Env {
+  account: string;
+  region: string;
+}
+
+export let testEnv = async (): Promise<Env> => {
+  const response = await new AWS.STS().getCallerIdentity().promise();
+
+  const ret: Env = {
+    account: response.Account!,
+    region: process.env.AWS_REGION ?? process.env.AWS_DEFAULT_REGION ?? 'us-east-1',
+  };
+
+  testEnv = () => Promise.resolve(ret);
+  return ret;
+};
+
+export let cloudFormation = makeAwsCaller(AWS.CloudFormation);
+export let sns = makeAwsCaller(AWS.SNS);
+export let iam = makeAwsCaller(AWS.IAM);
+export let lambda = makeAwsCaller(AWS.Lambda);
+export let sts = makeAwsCaller(AWS.STS);
```

Comment:
ES6+ would have imports be immutable. It's kinda frowned upon to assign to them directly. This means those should really better be `export const`. That said, it'd be easy enough to move all of this down into some mutable object that's exported, so I don't object leaving it as-is because it's nice and pragmatic.

Reply:
Yeh it should totally be 'export const', this is detritus.